tests/ui/uninlined_format_args.rs

```diff
@@ -211,12 +213,14 @@ macro_rules! my_println2_args {
     }};
 }
 
+#[clippy::format_args]
 macro_rules! my_concat {
     ($fmt:literal $(, $e:expr)*) => {
         println!(concat!("ERROR: ", $fmt), $($e,)*)
     }
 }
 
+#[clippy::format_args]
 macro_rules! my_good_macro {
     ($fmt:literal $(, $e:expr)* $(,)?) => {
```

Comment:
`#[clippy::format_args]` is applied to many of these macros but only `my_good_macro` is linting, is that intentional?